app/classifier/drawing-tools/ellipse.cjsx

```diff
@@ -53,6 +56,13 @@ module.exports = React.createClass
     x: @props.mark.rx * Math.cos theta
     y: -1 * @props.mark.ry * Math.sin theta
 
+  dragEnd: (target) ->
+    @context.geordi?.logEvent type: 'drag-ellipse'
+    deleteIfOutOfBounds target
+
+  logResize: ->
+    @context.geordi?.logEvent type: 'resize-ellipse'
+
```

Comment:
Can Geordi handle it? Maybe this event logging should be added later and in the interim do some performance testing?